Gems/ScriptCanvas/Code/Include/ScriptCanvas/Grammar/AbstractCodeModel.cpp

```diff
@@ -237,18 +237,17 @@ namespace ScriptCanvas
 
                 if (auto datum = variablePair.second.GetDatum())
                 {
-                    // #functions2 slot<->variable consider getting all variables from the UX variable manager, or from the ACM and looking them up in the variable manager for ordering
+                    // #sc_user_slot_variable_ux consider getting all variables from the UX variable manager, or from the ACM and looking them up in the variable manager for ordering
```

Comment:
All the #sc_user_slot_variable_ux comments need to be removed

Reply:
X